### ISSUE TITLE
Measure serialization and deserialization alone

### ISSUE THREAD

**Issue by giggio** [lazarofl/protocvsjsoncomparison#1]
Right not it is only showing serialization and deserialization together. I want to see this number, but I also want to see them alone

**Comment by lazarofl** [comment]
I agree with you, in fact, serialization and deserialization could be compared in isolation
